uro98/assignment1/Anagram.java

```diff
@@ -0,0 +1,72 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class Anagram {
+
+	public static Boolean wordAnagram(String a, String b, Boolean sensitive) {
+		if (!sensitive) {
+			a = a.toLowerCase();
+			b = b.toLowerCase();
+		}
+
+		HashMap<Character, Integer> hisA = new HashMap<Character, Integer>();
```

Comment:
Optional: variable types can be just `Map<Character, Integer>` since in code below you do not actually care about what kind of map is being used.